dist/spec.json

```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.9.1"
+    "version": "2.9.2"
```

Comment:
should this be a major rev instead?

Reply:
This should not be a breaking change for other SDKs. So IMHO it could be minor rev update